yarn-project/aztec-nr/aztec/src/utils.nr

```diff
@@ -17,6 +17,7 @@ pub fn full_field_greater_than(lhs: Field, rhs: Field) -> bool {
     rhs.lt(lhs)
 }
 
+// TODO(benesjan): also in circuits. NUKE THIS!
```

Comment:
This is a copy from aztec-nr. I will nuke the version in aztec-nr once I implement deserialize for callstack items as well which I didn't do in this PR because I already had #4381 open with a lot of changes there.